src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,120 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exceptions.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    public static final int GENERATED_KEY_INDEX = 1;
+    public static final int STATEMENT_FIRST_PARAM_INDEX = 1;
+    public static final int STATEMENT_SECOND_PARAM_INDEX = 2;
+    public static final int STATEMENT_THIRD_PARAM_INDEX = 3;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertQuery = "INSERT INTO manufacturers (name, country) VALUES (?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createStatement = connection
+                        .prepareStatement(insertQuery, Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(STATEMENT_FIRST_PARAM_INDEX, manufacturer.getName());
+            createStatement.setString(STATEMENT_SECOND_PARAM_INDEX, manufacturer.getCountry());
```

Comment:
не треба виносити це в костанти, це вже лишнє, і далі теж виправ
```suggestion
            createStatement.setString(1, manufacturer.getName());
            createStatement.setString(2, manufacturer.getCountry());
```